src/core/instance/render-helpers/render-slot.js

```diff
@@ -15,6 +15,12 @@ export function renderSlot (
   if (scopedSlotFn) { // scoped slot
     props = props || {}
     if (bindObject) {
+      if (!isObject(bindObject)) {
+        process.env.NODE_ENV !== 'production' && warn(
```

Comment:
can you move the `process.env.NODE_ENV !== 'production'` into the if condition so it gets stripped off in production mode?

Reply:
Oh, of course I can move it!
Sure, I will add the test.